hax/treemakers/corrections.py

```diff
@@ -53,6 +53,14 @@ class Corrections(TreeMaker):
 
     extra_metadata = hax.config['corrections_definitions']
 
+    #Check if data was generated with MC.
+    #If it was, pull the electron lifetime values from the associated metadata
+    def check_for_mc(self):
+        mc_data = False
+        if 'MC' in hax.paxroot.get_metadata(self.run_name)['configuration']:
+            mc_data = hax.paxroot.get_metadata(self.run_name)['configuration']['MC']['mc_generated_data']
+        return mc_data
+
     # Electron Lifetime: hopefully doc was pulled in hax.init.
     # Otherwise get it here at significantly higher DB cost
     try:
```

Comment:
Should this check be skipped for MC?

Reply:
I didn't want it to throw an error if it checks for MC data in the configuration settings and doesn't find it there